Sources/CMakeLists.txt

```diff
@@ -110,6 +113,7 @@ if(CMAKE_CXX_COMPILER_ID MATCHES "Clang")
     add_definitions(-Wno-switch)
     add_definitions(-Wno-tautological-undefined-compare)
     add_definitions(-Wno-c++11-compat-deprecated-writable-strings)
+    add_definitions(-Wno-logical-op-parentheses)
```

Comment:
These are actually legit bugs being reported, though, and I'd like them to be cleaned up at some point.